kaldo/observables/thirdorder.py

```diff
@@ -168,7 +168,8 @@ def load(cls, folder, supercell=(1, 1, 1), format='sparse', third_energy_thresho
             replicated_atom_prime_file = str(folder) + '/' + replicated_filename
             atoms = ase.io.read(atom_prime_file)
             replicated_atoms = ase.io.read(replicated_atom_prime_file)
-            with open('FC3', 'r') as f:
+            file_name = str(folder) + '/' + 'FC3'
+            with open(file_name, 'r') as f:
                 lines = f.readlines()
                 f.close()
             c = 13.605693012183622 * (1.889725989 ** (3))
```

Comment:
What are these values?
`c = 13.605693012183622 * (1.889725989 ** (3))`

Reply:
If you want to use Rydberg or other units, please import the ASE module.

Reply:
The following site might be helpful: https://wiki.fysik.dtu.dk/ase/ase/units.html
Alternatively, scipy also has functions to handle units/constants: https://docs.scipy.org/doc/scipy/reference/constants.html